yt_dlp/extractor/audiodraft.py

```diff
@@ -0,0 +1,93 @@
+from .common import InfoExtractor
+from ..utils import int_or_none
+
+
+class AudiodraftIE(InfoExtractor):
```

Comment:
The convention is for the same to end in `BaseIE`
```suggestion
class AudiodraftBaseIE(InfoExtractor):
```